pkg/kubelet/cm/topologymanager/topologymanager.go

```diff
@@ -164,8 +163,25 @@ func getTopologyAffinity(arrangedMask, maskHolder []string) []string {
 }
 
 func parseMask(mask []string) []int64 {
-	maskStr := mask[0]
-        glog.Infof("Mask string in ParseMask: %v", maskStr)
+	var maskStr string
+	var countSlice []int64
+	for i := 0; i < len(mask); i++ {

```

Comment:
Would it be possible to avoid three separate for loops?
min could be the count of the first element in the mask array, then check if num is lower than min. If it is then min = num and maskStr = mask[i]

Reply:
Yes, that makes far more sense! :)